api/project/models/project.settings.json

```diff
@@ -82,6 +82,11 @@
       "via": "related",
       "plugin": "upload",
       "required": false
+    },
+    "Draft_flag": {
+      "type": "boolean",
+      "default": true,
+      "private": true
```

Comment:
It shouldn't be private - we'll need it on the frontend to know which projects to display. It should, however, be required.